apps/src/netsim/NetSimSendPanel.js

```diff
@@ -412,6 +412,10 @@ NetSimSendPanel.prototype.onSendEventTriggered_ = function (jQueryEvent) {
   if (level.messageGranularity === MessageGranularity.PACKETS) {
     this.beginSendingPackets_();
   } else if (level.messageGranularity === MessageGranularity.BITS) {
+    if (this.getNextBit_() === undefined) {
+      // Ignore send events in BITS mode when there are no bits queued.
```

Comment:
Why not have this ignore down [below](#diff-67b570f2d54a98efd46a6a77d2e3f156R437) where we formerly threw an error?